Sites/GD2/Project 1/Styles.css

```
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
+}
+
+h1 {
+    color: #260648;
+    font-weight: bold;
+    font-size: 40px
+}
+
+bodycolor {
+    color: #F6F6EC
+}
+
+h2 {
+    color: #6B4B8C;
+    font-weight: bold;
+    font-size: 20px;
+    font-alignment: left
+}
+
+div {
+    margin: 12px
+}
+
+width=39em p {
+    color: #1C0534;
+}
+
+a {
+    color: #EE3595;
+}
+
+a:hover {
+    text-decoration: underline;
+
+    /* Will underline all links when the user puts their mouse over them */
+}
+
+p1 {
+    font-weight: regular;
```

You're looking for this value instead:
```suggestion
    font-weight: normal;
```